MANIFEST

```diff
@@ -61,6 +61,7 @@ keywords.h				The keyword numbers
 l1_char_class_tab.h			256 word bit table of character classes (for handy.h)
 locale.c				locale-specific utility functions
 locale_table.h				header for locale.c
+lock_definitions.h
```

Comment:
Can you provide a description for this file in the second column?  (And should we mention that this is a generated file?)

Reply:
fixed